### DISCUSSION TITLE
Use multiple ImageUpdateAutomations for different use cases

### DISCUSSION THREAD

**Discussion by ntx-ben** [fluxcd/flux2#4115] [Proposals] [answered]
I could not find any documentation or examples about this, but I'm looking into having two different `ImageUpdateAuotmation`s specs: 

- one for internal services where the commit is made directly into our develop branch, and thus deployed right away
- and another for 3rd party services we use where the commit is made into a separate, dedicated branch for PR to be created for review by the team.

Has anyone ran into the same kind of situation?

**Comment by stefanprodan** [comment, marked as answer]
The `ImageUpdateAuotmation` acts on all polices in a namespace, if the 3rd party services are in a different namespace than the apps, you can use different automations.
